App/.env

```diff
@@ -0,0 +1,11 @@
+AAD_CLOUD_INSTANCE=Enter_the_Cloud_Instance_Id_Here # cloud instance string should end with a trailing slash
+AAD_TENANT_INFO=Enter_the_Tenant_Info_Here
+AAD_CLIENT_ID=Enter_the_Application_Id_Here
+AAD_CLIENT_SECRET=Enter_the_Client_Secret_Here
+
+AAD_REDIRECT_URI=http://localhost:3000/auth/redirect
+AAD_POST_LOGOUT_REDIRECT_URI=http://localhost:3000
+
+GRAPH_API_ENDPOINT=Enter_the_Graph_Endpoint_Here # graph api endpoint string should end with a trailing slash
```

Comment:
>Enter_the_Graph_Endpoint_Here # graph api endpoint string should end with a trailing slash [](http://example.com/codeflow?start=19&length=90)

I'd suggest we place "[https://graph.microsoft.com/](https://graph.microsoft.com/)" as default and comments about changing it for other instances of Graph

Reply:
We can't, portal automation replaces this with the corresponding Graph API instance (national deployments etc)